HelpSource/Classes/Process.schelp

```diff
@@ -56,3 +58,235 @@ list::
 This means that link::Classes/Thread#.thisThread#thisThread:: will always initially point
 to the main Thread. However, when some code starts a link::Classes/Routine::, the Routine
 becomes the current Thread, with the main Thread as its parent.
+
+examples::
+
+strong::Example 1.:: Comparison of the path of the evaluated code block in a saved SCD, the loaded SCD and the function in the loaded SCD:
+
+table::
+##
+
+If a code block ("fileMain.scd" in the example code below) executes another file on disk ("fileForLoad.scd" in the example code below), using link::Classes/String#-load:: or link::Classes/String#-loadPaths::, teletype::thisProcess.nowExecutingPath:: will be the location of the executed file ("fileForLoad.scd" in the example code below). However, if the function is called in the executed file ("fileForLoad.scd" in the example code below), teletype::thisProcess.nowExecutingPath:: will be the document of the code block ("fileMain.scd" in the example code below).
+
+strong::Steps:::
+
+numberedlist::
+
+## Preparation code:
+
+code::
+(
+var test, fileMain, fileForLoad;
+
+test = "thisProcess.nowExecutingPath";
+~fileMainPath = "~/fileMain.scd".standardizePath;
+fileMain = File(~fileMainPath, "w");
+fileMain.write(
+	"(\n" ++
+	"(" ++ (test ++ ":").quote ++ ").postln;\n" ++
+	"(" ++ ("\tin fileMain.scd:                       ").quote + "+" + test ++ ").postln;\n" ++
+	"~test = (" ++ test ++ ".dirname +/+" + "fileForLoad.scd".quote ++");\n" ++
+	"~test.load.testFunction;\n" ++
+	"~test.openOS; // n.b.: .openDocument only works in SC-IDE\n" ++
```

Comment:
Maybe just avoid code like .openOS that only works in scide?

Reply:
`openOS` works in the terminal and the SC-IDE. Does it not work on Emacs?

Reply:
> `openOS` works in the terminal and the SC-IDE. Does it not work on Emacs?

It does something strange. If I evaluate this in Emacs, it opens SCIDE ...

Reply:
I would use this method for files like PDF, but it can result in strange behaviors with scd files.

Reply:
This is because SC-IDE is the default application for scd document on your OS.

Reply:
I would just describe the code in each file. Maybe you don't need to create two files to show.

Reply:
Yes, I usually have emacs running first and open files from it. So I don't change xdg-open default.

Reply:
I also do not close SC-IDE. Anyway I changed that part as follows:
```
code::
 ~fileMainPath.openOS;
 // Note: .openDocument only works in SC-IDE.
 // .openOS will work in other editors and the system default application will open the SCD file.
 ::
```

Reply:
Can I mark it as solved?
 
![Screenshot 2024-05-10 at 08 54 23](https://github.com/supercollider/supercollider/assets/416281/d22325c7-5015-4c92-a94e-d405431af6f6)